sjb/config/common/test_cases/crio.yml

```diff
@@ -108,10 +108,10 @@ post_actions:
       then
           sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
           sudo yum install -y jq docker
+          sudo systemctl start docker
```

Comment:
Would `systemctl enable --now docker` be better here?

Reply:
yes thanks, that is better, I'll add a patch for it